js/layout-javascript/news-feed-code.js

```diff
@@ -833,6 +868,29 @@ DynamicList.prototype.removeListItemHTML = function (options) {
   this.$container.find('.news-feed-list-item[data-entry-id="' + id + '"]').remove();
 }
 
+DynamicList.prototype.resetSortIcons = function() {
```

Comment:
Add a function to deal with this:

```js
this.Utils.UI.resetSortIcons = function (options) { }
```

...where `options` takes the property:

- `$sortList`

e.g.
```js
this.Utils.UI.resetSortIcons({ $sortList: this.container.find('.sort-list-news') })
```

It also means:

1. Don't use `#sort-list-news`, use `.sort-list-news`
1. Use `this.container.find()` instead of `$()`

Reply:
@YaroslavOvdii Why is this kept here?